### PR TITLE
add release notes 5.1

### PR DESCRIPTION
**Pull request by dcs3spp**
Add release notes 5.1 with change log, links and docker images

[preview](https://deploy-preview-2765--tyk-docs.netlify.app/docs/nightly/release-notes/)

### CODE REVIEW

File: tyk-docs/content/release-notes/version-5.1.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  main:
+    parent: "Release Notes"
+weight: 1
+---
+
+# What’s Changed?
+
+### Tyk Gateway and Dashboard updated to Golang version 1.19
+
+Our Dashboard and Gateway are using [Golang 1.19](https://tip.golang.org/doc/go1.19) Programming Language starting with the 5.1 release. This brings improvements to the code base and allows us to benefit from the latest features and security enhancements in Go. Don’t forget that, if you’re using GoPlugins, you’ll need to recompile these to maintain compatibility with the latest [Gateway](https://tyk.io/docs/upgrading-tyk/#tyk-go-plugins). Further information is also available [here]({{< ref "plugins/supported-languages/golang#initialise-plugin-for-gateway-51" >}})
```
Comment (letzya) [inline tyk-docs/content/release-notes/version-5.1.md]: bug in the links
```suggestion
Our Dashboard and Gateway are using [Golang 1.19](https://tip.golang.org/doc/go1.19) Programming Language starting with the 5.1 release. This brings improvements to the code base and allows us to benefit from the latest features and security enhancements in Go. Don’t forget that, if you’re using GoPlugins, you’ll need to recompile these to maintain compatibility with the latest [Gateway](upgrading-tyk#tyk-go-plugins). Further information is also available [here]({{< ref "plugins/supported-languages/golang#initialise-plugin-for-gateway-51" >}})
```

File: tyk-docs/content/release-notes/version-5.1.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+When we implemented Role Based Access Control and API Ownership in Tyk
+Dashboard, we unlocked great flexibility for you to assign different roles to
+different users and user groups with visibility and control over different
+collections of APIs on your Gateway. Well, from 5.1 we have added a new Role,
+which layers on top of the existing “Analytics” role and can be used to restrict
+a user’s access, within the Dashboard Analytics screens, to view only the
+statistics from APIs that they own; we’ve called this “Owned Analytics”. Due to
+the way the analytics data are aggregated (to optimise storage), a user granted
+this role will not have access to the full range of charts. Take a look at the
+documentation for a full description of this new [user role](https://tyk.io/docs/basic-config-and-security/security/dashboard/user-roles/).
```
Comment (letzya) [inline tyk-docs/content/release-notes/version-5.1.md]:
```suggestion
documentation for a full description of this new [user role]({{< ref "basic-config-and-security/security/dashboard/user-roles/" >}}).
```

**Review by letzya — DISMISSED** [review]

File: tyk-docs/content/release-notes/version-5.1.md (hunk 1 above)
Comment (dcs3spp) [reply tyk-docs/content/release-notes/version-5.1.md]: done

File: tyk-docs/content/release-notes/version-5.1.md (hunk 2 above)
Comment (dcs3spp) [reply tyk-docs/content/release-notes/version-5.1.md]: done

**Pull request merged**